File: 01_Language/01_Functions/php/fileatime.php
<?php

// outputs e.g.  somefile.txt was last accessed: December 29 2002 22:16:23.

$filename = 'test.txt';
if (file_exists($filename)) {
    echo "$filename was last accessed: " . date("F d Y H:i:s.", fileatime($filename));
}

File: 01_Language/01_Functions/python/fileatime.py
# coding: utf-8

import os
import time


def fileatime(filename):
    return os.path.getatime(filename)


if __name__ == '__main__':
    print(time.strftime("%Y-%m-%d %H:%M:%S", time.localtime(fileatime("test.txt"))))
